src/test/java/com/application/bookingservice/controller/BookingControllerTest.java

```diff
@@ -0,0 +1,108 @@
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+@Sql(scripts = {"classpath:database/booking/insert-addresses-table.sql",
+        "classpath:database/booking/insert-accommodations-table.sql",
+        "classpath:database/booking/insert-bookings-table.sql"},
+        executionPhase = Sql.ExecutionPhase.BEFORE_TEST_CLASS)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class BookingControllerTest {
+    protected static MockMvc mockMvc;
+    private static final String MY_BOOKING_API = "/bookings/my";
+    private static final Long ACCOMMODATION_ID = 1L;
+    private static final LocalDate BOOKING_CHECK_OUT = LocalDate.of(2024, 12, 24);
+    private static final LocalDate BOOKING_CHECK_IN = LocalDate.of(2024, 12, 19);
+    private static final Long BOOKING_ID = 1L;
+    private static final Long CUSTOMER_ID = 1L;
+    private static final String CUSTOMER_EMAIL = "coolemail@gmai.com";
+    private static final String CUSTOMER_PASSWORD = "hardpassword";
+    private static final Long ROLE_ID = 1L;
+    private static List<BookingResponseDto> bookingResponseDtos;
+    @Autowired
+    private ObjectMapper objectMapper;
+
+    @BeforeAll
+    static void beforeAll(
+            @Autowired WebApplicationContext applicationContext
+    ) {
+        mockMvc = MockMvcBuilders
+                .webAppContextSetup(applicationContext)
+                .apply(springSecurity())
+                .build();
+        BookingResponseDto booking = new BookingResponseDto();
+        booking.setAccommodationId(ACCOMMODATION_ID);
+        booking.setCheckOut(BOOKING_CHECK_OUT);
+        booking.setCheckIn(BOOKING_CHECK_IN);
+        booking.setStatus(PENDING);
+        booking.setId(BOOKING_ID);
+        booking.setCustomerId(CUSTOMER_ID);
+        bookingResponseDtos = List.of(booking);
+    }
+
+    @BeforeEach
+     void setUp() {
+        Customer customer = new Customer();
+        customer.setId(CUSTOMER_ID);
+        customer.setEmail(CUSTOMER_EMAIL);
+        customer.setPassword(CUSTOMER_PASSWORD);
+        Role role = new Role();
+        role.setId(ROLE_ID);
+        role.setRole(Role.RoleName.ROLE_CUSTOMER);
+        customer.setRoles(Set.of(role));
+        Authentication authentication
+                = new UsernamePasswordAuthenticationToken(customer,
+                customer.getPassword(), customer.getAuthorities());
+        SecurityContextHolder.getContext().setAuthentication(authentication);
+    }
+
+    @WithMockUser(username = "Customer", roles = {"CUSTOMER"})
+    @Test
+    @DisplayName("""
+            verify get my booking work
+            """)
+    public void getAll_ValidData_Success() throws Exception {
+        List<BookingResponseDto> expected = new ArrayList<>(bookingResponseDtos);
```

Comment:
you have structure: expected -> result -> actual -> assertions
follow this structure please: mvcResult ->(empty line) expected ->(no empty line) actual ->(empty line) assertions